### PR TITLE
Merged and updated body file entries

### PR DESCRIPTION
**Pull request by bromiley**
TSK documentation uses 'body file' as two separate words, so I went with that as the default. `bodyfile` points to `body file`. Changes/improvements made:

 - Merged and updated articles
 - Consolidated links and formatting
 - Consistent Markdown formatting throughout

### CODE REVIEW

File: docs/body_file.md [hunk 1]
```diff
@@ -8,138 +8,156 @@ tags:
...
+# Body File
 
-    MD5 | path/name | device | inode | mode_as_value | mode_as_string | num_of_links | UID | GID | rdev | size | atime | mtime | ctime | block_size | num_of_blocks
+_Body file may also be referred to as "bodyfile", however official documentation refers to it as body file (two separate words)._
 
-Version 3 (used by SleuhtKit 3 and later):
+The **body file** format is a delimiter-separated output timeline format introduced by the [SleuthKit](sleuthkit.md). Body files are pipe (`|`) delimited and are referred to as an "intermediate file", as they are not sorted chronologically and are often staged for post-processing. Subsequent timeline sorting is done via the [mactime](mactime.md) tool.
 
-    MD5|name|inode|mode_as_string|UID|GID|size|atime|mtime|ctime|crtime
+Body files are often generated by the following tools:
```
Comment (joachimmetz) [inline docs/body_file.md:18]: there are many other tools that generate some form of body files, I would leave this out of the text.

File: docs/body_file.md [hunk 2]
```diff
@@ -8,138 +8,156 @@ tags:
...
+<!-- Need to insert links to these tools -->
+* fls
+* ils
 
-The times are reported in a POSIX (or UNIX) timestamp.
+## Data & Fields
 
-### Shortcomings
+All times within a body file are reported in UNIX time format. Lines that start with `#` are ignored and treated as comments.
```
Comment (joachimmetz) [inline docs/body_file.md:25]: `Lines that start with `#` are ignored and treated as comments.` do you have a reference to this?

File: docs/body_file.md (hunk 1 above)
Comment (joachimmetz) [reply docs/body_file.md:18]: I see that is mentioned here https://wiki.sleuthkit.org/index.php?title=Body_file

File: docs/body_file.md (hunk 2 above)
Comment (joachimmetz) [reply docs/body_file.md:25]: I see that is mentioned here https://wiki.sleuthkit.org/index.php?title=Body_file

File: docs/body_file.md [hunk 3]
```diff
@@ -8,138 +8,156 @@ tags:
...
+---
+
+## Known Issues
+<!-- Issues have been merged from multiple files. Also, issues may apply to The Sleuth Kit more than body files - must reconcile. 
+NOTE: Nov/Dec 2022, validate if each is still an issue -->
+Known shortcomings with body file format are:
+
+* Undocumented granularity of timestamp, current implementation by
+  [The Sleuth Kit](sleuthkit.md) appears to be seconds. See [here](https://github.com/sleuthkit/sleuthkit/issues/1810).
+* Undocumented extended file mode of `-/-rwxrwxrwx`.
```
Comment (joachimmetz) [inline docs/body_file.md:54]: maybe reword this to make it clear that both `-/-` and the leading `r` in `rrwxrwxrwx` are undocumented

File: docs/body_file.md [hunk 4]
```diff
@@ -8,138 +8,156 @@ tags:
...
+* Undocumented and inconsistent application of TSK metadata addresses. See [here](https://github.com/sleuthkit/sleuthkit/issues/1809)
+* Undocumented and inconsistent application of owner UID. See here.[here](https://github.com/sleuthkit/sleuthkit/issues/1830)
```
Comment (joachimmetz) [inline docs/body_file.md:56]: Consolidate `here.[here]` into `[here]`

File: docs/body_file.md [hunk 5]
```diff
@@ -8,138 +8,156 @@ tags:
...
+* Date and time values do not indicate a time zone or if daylight savings applies. All timestamps are assumed to be in UTC.
+* Body file encoding is not specified, UTF-8 is assumed.
+* The name field can contain (`$FILE_NAME`) to indicate the bodyfile entry was derived from a NTFS `$FILE_NAME` attribute instead of `$STANDARD_INFORMATION` and `$DATA` attributes. Note that the exact behavior is not documented by the SleuthKit project.
```
Comment (joachimmetz) [inline docs/body_file.md:59]: bodyfile => body file

File: docs/body_file.md [hunk 6]
```diff
@@ -8,138 +8,156 @@ tags:
...
+* The `name` field can contain `-\> symbolic_link_target` but `fls` does not appear to support this for NTFS. Also see [here](https://github.com/sleuthkit/sleuthkit/issues/2645).
+* It is unclear if the symbolic link target can be used in combination with the `$FILE_NAME` suffix.
+* It is unclear which characters should be escaped, by observation `|`
+  and `\` are both escaped with `\` in the name field by `fls` but `mactime`
+  is unable to handle a name that contains the `|` character. See [here](https://github.com/sleuthkit/sleuthkit/issues/2124). _Note: Other implementations are known to not escape `\`._
```
Comment (joachimmetz) [inline docs/body_file.md:64]: consistency nit: `_Note` on this line is italic but not on other lines

File: docs/body_file.md [hunk 7]
```diff
@@ -8,138 +8,156 @@ tags:
...
+* The format of the `mode_as_string` field is unclear for file systems like NTFS, this likely can be derived from the source code. Also see [here](https://github.com/sleuthkit/sleuthkit/issues/1810).
+* Note that the SleuthKit currently does not correctly identifies symbolic links for NTFS in the body file output. Also see [here](https://github.com/sleuthkit/sleuthkit/issues/2645).
+* The `atime`, `mtime`, `ctime`, and `crtime` typically contain the number of seconds since January 1, 1970. It is unknown if a fractional part is allowed by specification. The corresponding mactime tool does allow for a fractional part to be present but ignores it. Also see [here](https://github.com/sleuthkit/sleuthkit/issues/1810>). _This limits the usefulness of the format for timelines with a vast amount of sub-second activity._
```
Comment (joachimmetz) [inline docs/body_file.md:70]: `https://github.com/sleuthkit/sleuthkit/issues/1810>` remove the trailing `>`

File: docs/body_file.md [hunk 8]
```diff
@@ -8,138 +8,156 @@ tags:
...
+```
 
 Behavior of the TSK Metadata Address:
 
-- 11-144-4 references the \$INDEX_ROOT attribute in MFT entry 11, note
-  that this \$INDEX_ROOT attribute on-disk is the 3rd attribute in the
-  MFT entry. Note that the size of a corresponding \$INDEX_ALLOCATION is
-  not included in the "size" in the body file.
-  [4](https://github.com/sleuthkit/sleuthkit/issues/1831)
+* `8-128-2` references the `$DATA` attribute in MFT entry 8, note that this
```
Comment (joachimmetz) [inline docs/body_file.md:147]: Add a note that this breaks down when there is an ATTRIBUTE_LIST since it is not deterministic what the "first" MFT attribute is.

**Review by joachimmetz — COMMENTED** [review]
Left a couple of nits and highlighted some small formatting issues.

**Comment by joachimmetz** [comment]
@bromiley thanks for updating these pages

File: docs/body_file.md [hunk 9]
```diff
@@ -8,138 +8,156 @@ tags:
...
-[SleuthKit](sleuthkit.md).
 
-Version 2 (used by SleuhtKit 2):
+# Body File
 
-    MD5 | path/name | device | inode | mode_as_value | mode_as_string | num_of_links | UID | GID | rdev | size | atime | mtime | ctime | block_size | num_of_blocks
+_Body file may also be referred to as "bodyfile", however official documentation refers to it as body file (two separate words)._
 
-Version 3 (used by SleuhtKit 3 and later):
+The **body file** format is a delimiter-separated output timeline format introduced by the [SleuthKit](sleuthkit.md). Body files are pipe (`|`) delimited and are referred to as an "intermediate file", as they are not sorted chronologically and are often staged for post-processing. Subsequent timeline sorting is done via the [mactime](mactime.md) tool.
```
Comment (joachimmetz) [inline docs/body_file.md:16]: Add `(as far known) introduced` since the format might originate from other tools

From https://github.com/sleuthkit/sleuthkit/blob/develop/tools/timeline/mactime.base 
```
# This program is based on the 'mactime' program by Dan Farmer and
# and the 'mac_daddy' program by Rob Lee.
```

Comment (bromiley) [reply docs/body_file.md:16]: Added in.

File: docs/body_file.md (hunk 6 above)
Comment (bromiley) [reply docs/body_file.md:64]: Removed as "Note", and instead made a major bullet.

File: docs/body_file.md (hunk 8 above)
Comment (bromiley) [reply docs/body_file.md:147]: Addressed in commit.

**Review by joachimmetz — APPROVED** [review]
LGTM
